lighthouse-core/report/html/report-styles.css

```diff
@@ -593,13 +588,15 @@
 
 .lh-metric__description {
   display: none;
+  grid-column-start: 1;
+  grid-column-end: 3;
```

Comment:
looking at https://yuinchien.com/sandbox/lighthouse/
![image](https://user-images.githubusercontent.com/39191/60309195-72b51d80-9901-11e9-81a1-80365596a890.png)

it seems this is currently misaligned...
![image](https://user-images.githubusercontent.com/39191/60309199-79dc2b80-9901-11e9-8aff-1754728b76a0.png)



perhaps:

```css
  grid-column-start: 2;
  grid-column-end: 4;
```

? i dont really know grid though... can you doublecheck?

Reply:
ya that did the trick